config/daemon/systemd.md

```diff
@@ -77,6 +77,11 @@ you need to add this configuration in the Docker systemd service file.
     ```bash
     $ sudo mkdir -p /etc/systemd/system/docker.service.d
     ```
+     
+    Note: for [rootless](https://docs.docker.com/engine/security/rootless/) docker:
+    ```bash
+    $ mdkir -p ~/.config/systemd/user/docker.service.d
```

Comment:
typo

Reply:
Thanks for reviewing @AkihiroSuda, however I'm not sure where the typo is? Are you referring to the directory name `docker.service.d`. 

Systemctl correctly recognizes the directory and the drop-in config file inside it, please see snapshot below:
```
$ systemctl --user status docker
● docker.service - Docker Application Container Engine (Rootless)
   Loaded: loaded (/home/xxxxxx/.config/systemd/user/docker.service; disabled; vendor preset: enabled)
  Drop-In: /home/xxxxxx/.config/systemd/user/docker.service.d
           └─http-proxy.conf
   Active: active (running) since Wed 2020-02-12 09:15:08 NZDT; 1h 3min ago
     Docs: https://docs.docker.com
 Main PID: 46025 (rootlesskit)
```

Reply:
mdkir